src/appleseed/renderer/kernel/lighting/lighttree.cpp

```diff
@@ -199,11 +213,15 @@ void LightTree::draw_tree_structure(
 }
 
 //
-// Calculate the tree depth and assign total luminance to each node of the tree.
-// Total luminance represents the sum of all its child nodes luminances.
-//
-
-float LightTree::recursive_node_update(size_t node_index, size_t node_level)
+// Calculate the tree depth.
```

Comment:
Comments describing the purpose of a method must go next to the method declaration (and not be repeated at the place of implementation).